src/Module/Admin/Logs/View.php

```diff
@@ -81,13 +80,35 @@ public static function content(array $parameters = [])
 			}
 		}
 		return Renderer::replaceMacros($t, [
-			'$title'         => DI::l10n()->t('Administration'),
-			'$page'          => DI::l10n()->t('View Logs'),
+			'$title' => DI::l10n()->t('Administration'),
+			'$page'  => DI::l10n()->t('View Logs'),
+			'$l10n'  => [
```

Comment:
Hmmm, interesting.

Reply:
This results more readable source code. I would like to see this solution for other template files.

Reply:
Thank you for your input, this is something you would be able to help with yourself. Template files are mentioned in the PHP source code where the variables are passed, it just is a matter of isolating the translation strings in the variable array and updating the references in the template files (watch our for theme-specific copies of the same template name).